esphome/components/rf_bridge/rf_bridge.h

```diff
@@ -165,5 +178,16 @@ template<typename... Ts> class RFBridgeSendRawAction : public Action<Ts...> {
   RFBridgeComponent *parent_;
 };
 
+template<typename... Ts> class RFBridgeBeepAction : public Action<Ts...> {
+ public:
+  RFBridgeBeepAction(RFBridgeComponent *parent) : parent_(parent) {}
+  TEMPLATABLE_VALUE(uint16_t, raw)
+
+  void play(Ts... x) { this->parent_->beep(this->raw_.value(x...)); }
```

Comment:
Please rename the variable (and references)
```suggestion
  TEMPLATABLE_VALUE(uint16_t, duration)

  void play(Ts... x) { this->parent_->beep(this->duration_.value(x...)); }
```